src/transformers/keras_callbacks.py

```diff
@@ -98,6 +114,16 @@ def on_epoch_end(self, epoch, logs=None):
             if self.checkpoint:
                 checkpoint_dir = os.path.join(self.output_dir, "checkpoint")
                 self.model._save_checkpoint(checkpoint_dir, epoch)
+            # TODO Is there a better choice for model_name here?
+            train_summary = TrainingSummary.from_keras(
+                model=self.model,
+                model_name=self.hub_model_id,
+                keras_history=self.training_history,
+                **self.model_card_args,
+            )
+            model_card = train_summary.to_model_card()
+            print("Model card:")
+            print(model_card)
```

Comment:
Should be removed :-)

Reply:
Ah, sorry! The print statements were just for debugging, that's of course going to be changed to writing to an output file instead.

Reply:
Fixed!